shuup_tests/xtheme/test_resources.py

```diff
@@ -38,7 +38,10 @@ def render(self, context):
         return self.message
 
 
+@pytest.mark.skip()
 def test_resources():
+    # TODO: Give some reason why this test should pass. Why we should
```

Comment:
what's this?

Reply:
Test is broken since we don't render placeholders if not editable.

Reply:
I guess point mainly is that should we just remove the test or change it somehow or what?

Reply:
got it. if the user cases doesn't exist, there is no reason for this test then.